### ISSUE TITLE
cannot delete the resource in eXide 2.4.5

### ISSUE THREAD

**Issue by jesselet** [eXist-db/eXide#192]
1. OK in the XQuery
 ```
 xmldb:store("/db/apps", "test1.xml", <test/>),
 xmldb:remove("/db/apps", "test1.xml")
 ```
2. Fail to delete the resource '/db/apps/test1.xml' in eXide 2.4.5 (Menu: File/Manage)
 in the __exist.log__:
 exerr:ERROR Resource or collection '__/db/apps__**/db/apps**/test1.xml' does not exist.

**Comment by joewiz** [comment]
Could you please clarify, what are the exact steps to reproduce this failure in eXide?

**Comment by duncdrum** [comment]
tested on exist-db: `5.0.0-4b1e9b359` and store and remove works as expected, no errors in log. 

@jesselet what exist, java, and os versions are you doing this in.

**Comment by jesselet** [comment]
OS: Win7 SP1 64-bit
Java: JRE 1.8.0_171
Browser: Chrome 67.0.3396.79

eXist | shared-resources | eXide | delete from eXide
------|------------------|-------|-------------------
3.6.1 | 0.4.2            | 2.4.2 | OK
3.6.1 | 0.4.2            | 2.4.5 | Fail
3.6.1 | 0.7.0            | 2.4.5 | Fail
4.2.0 | 0.5.0            | 2.4.5 | Fail
4.2.0 | 0.7.0            | 2.4.5 | Fail

**Comment by joewiz** [comment]
Steps to reproduce?

**Comment by duncdrum** [comment]
copied the listing as is from the op into eXide, click evaluate, 
result: 
`2018-06-11 23:46:56,955 [qtp715911316-36] INFO  (NativeBroker.java [removeXMLResource]:2680) - Removing document test1.xml (754251) …`
as expected the file is written into db and immediately removed. 

macOS `10.13.5` 
java `1.8.0_172` 
exist `4.2.0` 
shared-resource `0.7.0`
exide `2.4.5`

I suspect this to be os related, since i can't reproduce this, and path handling is OS related. 
Someone with access to win pls take a look. Should there be a bug, we need to check why our win tests are not picking it up [xmldb:store](https://ci.appveyor.com/project/AdamRetter/exist/build/develop-4.x.x-675/job/wjlwpy9ulp7k8oor/tests)

**Comment by joewiz** [comment]
No, he was saying the query worked and successfully deleted the file. What did not work was deleting via File > Manage. I was asking for these steps!

**Comment by duncdrum** [comment]
@joewiz I did not get that from the description in the op, @jesselet please take the time to provide all the information necessary for us to reproduce, so we don't waste time chasing ghosts.

Yes I can reproduce this now, same error as the op.

**Comment by joewiz** [comment]
Steps to reproduce:

1. Create resource via `xmldb:store("/db/apps", "test1.xml", <test/>)`.
2. Open DB Manager pane via File > Manage.
3. Navigate to /db/apps and select the test file.
4. Select delete.
5. Error message: 
    > Deletion of the following items failed: /db/apps/test1.xml.
6. Refresh DB Manager and notice the file is still there.
7. Repeat by creating two resources, test1.xml and test2.xml. Similar failure.

I fixed the behavior in #193. Reviewing the history of the source code, this bug had to be present for many years, so I'm surprised it wasn't reported earlier. Thank you for reporting it @jesselet. In future, I think you can see why we're asking for steps like those above to reproduce - to give us the best chance of seeing exactly what you are.

**Comment by joewiz** [comment]
Fixed in eXide v2.4.6, which is now available via the Package Manager - or download directly from the http://demo.exist-db.org/exist/apps/public-repo/packages/eXide.html?eXist-db-min-version=3.3.0-SNAPSHOT or https://github.com/eXist-db/eXide/releases.
